docs/walkthrough-1.7.md

```diff
@@ -0,0 +1,6 @@
+# Service Catalog Demonstration Walkthrough
+
+This document assumes that you've installed Service Catalog onto your cluster.
+If you haven't, please see [install-1.7.md](./install-1.7.md).
+
+TODO
```

Comment:
I'd write here that they should be looking at the 1.6 walkthrough doc, and link to it.

Reply:
The 1.6 doc won't work for them, so they shouldn't look there.

Reply:
Does the walkthrough not work at all? Or is it just that the service catalog is behind the aggregator, so there is no "service-catalog" k8s context. If the latter, it seems misleading to basically send the reader to a dead end.

Reply:
Correct, there is no `service-catalog` k8s context. It doesn't work at all.

@jboyd01 and I are going to follow-up immediately after this PR to fill in this section.